esp/esp/cache/apps.py

```diff
@@ -1,14 +1,10 @@
-from django.apps import AppConfig
+from argcache.apps import ArgCacheConfig
 
-class CacheConfig(AppConfig):
+class ESPCacheConfig(ArgCacheConfig):
```

Comment:
This whole file can just get replaced by uses of the caches.py thing you added in the new one, right?